### PR TITLE
rm nexus sdk usage and dependency

### PR DESCRIPTION
**Pull request by ssssarah**
I've put all the code from the sdk that was used in the file previous_sdk_usage, with the change that the endpoint and token are provided as parameters (as opposed to setting environment variables)

No design thought really went into this, it's a fast solution, up to change

(preparatory work so that we specify a token as a callable, without having to handle setting the nexussdk environment variable)

### CODE REVIEW

File: kgforge/specializations/stores/bluebrain_nexus.py
Comment (lecriste) [inline kgforge/specializations/stores/bluebrain_nexus.py:1]: Shall we have a test for the function `_upload_one` changed here?

Comment (ssssarah) [reply kgforge/specializations/stores/bluebrain_nexus.py:1]: Sure, but we don't have integration tests in this repository so I wouldn't know how to test it?

**Review by lecriste — APPROVED** [review]

File: kgforge/specializations/stores/bluebrain_nexus.py
Comment (crisely09) [reply kgforge/specializations/stores/bluebrain_nexus.py:1]: This is sort of "tested" with the notebooks, the files are used in the getting started notebooks. Just check if they run correctly.

File: kgforge/specializations/stores/nexus/previous_sdk_usage.py [hunk 1]
```diff
@@ -0,0 +1,297 @@
+import puremagic
```
Comment (crisely09) [inline kgforge/specializations/stores/nexus/previous_sdk_usage.py:1]: what are the differences with mime types?

**Comment by crisely09** [comment]
hmm why to call things: `previous_sdk_usage.py` ? why not just something generic about the methods there? and just comment/docstring that this was previois sdk usage

File: kgforge/specializations/stores/bluebrain_nexus.py
Comment (crisely09) [reply kgforge/specializations/stores/bluebrain_nexus.py:1]: I checked, notebooks run fine.

**Pull request merged**